pdh_windows.go

```diff
@@ -0,0 +1,120 @@
+package sigar
+
+/* These are convenience methods for extracting metrics from the Windows
+   Performance Data Helper (PDH) library */
+
+import (
+	"fmt"
+	"github.com/scalingdata/win"
+	"unsafe"
```

Comment:
It feels kind of weird that we have a wrapper for accessing Windows C api functions, but we still end up using the `unsafe` package.